### PR TITLE
Custom labels for ServiceMonitor resource

### PR DESCRIPTION
**Pull request by nlamirault**

### CODE REVIEW

File: charts/descheduler/templates/servicemonitor.yaml [hunk 1]
```diff
@@ -7,6 +7,9 @@ metadata:
   namespace: {{ .Values.serviceMonitor.namespace | default .Release.Namespace }}
   labels:
     {{- include "descheduler.labels" . | nindent 4 }}
+    {{- if .Values.serviceMonitor.additionalLabels }}
+    {{ toYaml .Values.serviceMonitor.additionalLabels }}
```
Comment (a7i) [inline charts/descheduler/templates/servicemonitor.yaml:11]: shouldn't this be indented?
```suggestion
    {{ toYaml .Values.serviceMonitor.additionalLabels | nindent 4 }}
```

**Review by damemi — COMMENTED** [review]
/approve

**Pull request merged**